tests/unit/executor_test_base.py

```diff
@@ -109,6 +109,8 @@ def clear_db(self, db):
         db.session.add(r)
         r = db.Integration(name='lightwood', data={}, engine='lightwood')
         db.session.add(r)
+        r = db.Integration(name='fbprophet', data={}, engine='fbprophet')
+        db.session.add(r)
         db.session.flush()
         self.lw_integration_id = r.id
```

Comment:
lw_integration_id should point at lightwood integration.
this line should be moved where lightwood integration is creating

Reply:
@yuhuishi-convect can you please swap this around so that the last integration to be added is Lightwood? It seems it breaks the CI otherwise.